scripts/module/keybinder.nut

```diff
@@ -1,34 +1,38 @@
+function newBindKey( keycode ) {
+  return BindKey(true, keycode, 0, 0);
+}
```

Comment:
I assume this leads to excessive network traffic and security issues. We should only bind keys when we actually need them.

Reply:
I agree that binding all keys is an ugly hack. Binding all keys was done in order to simplify keypress comparison - especially as the resulting values of `BindKey()` are undocumented as far as I can tell.

Reply:
This could probably be restructured to use client side keybinds with sockets communication.
`onPlayerCommand()` is not available client side so you'd have to send command from client, receive at server, send back to client unmodified, do most of the same logic now implemented in server client side and then handle the occasional `Server.SendData()`stream sent string from `KeyBind::OnDown()` serverside. Why did I not structure it this way from the get-go? :woman_facepalming:

Best of all; These mentioned functions are half-documented as best on the wiki.